lib/adapters/http/index.js

```diff
@@ -950,7 +950,7 @@ function HttpPouch(opts, callback) {
             if (returnDocs) {
               results.results.push(c);
             }
-            utils.call(opts.onChange, c);
+            opts.onChange(c);
```

Comment:
Oh yeah, that was me who removed a bunch of dead code. But I guess after removing the dead code, I didn't notice that the whole thing was dead. XD